farm/data_handler/utils.py

```diff
@@ -64,11 +64,12 @@ def read_tsv(filename, rename_columns, quotechar='"', delimiter="\t", skiprows=N
     # let's rename our target columns to the default names FARM expects:
     # "text": contains the text
     # "text_classification_label": contains a label for text classification
-    columns = ["text"] + list(rename_columns.keys())
+    columns = list(rename_columns.keys())
```

Comment:
The code that was already present in FARM to rename columns of a pandas DF is a shame : ) and should be changed.

While we are at it: could you use the pandas functionality for renaming the columns here, please? 
`df.rename(columns=rename_columns, inplace=True)`

Reply:
I made the changes.